src/query/functions/temporal/aggregation.go

```diff
@@ -49,6 +49,9 @@ const (
 
 	// StdVarType calculates the standard variance of all values in the specified interval
 	StdVarType = "stdvar_over_time"
+
+	// HoltWintersType produces a smoothed value for time series based on the specified interval
```

Comment:
nit: I think we're slowly moving to having `.` at the end of comments... can probably keep it for a later PR so this doesn't get filled with crap though

Reply:
Also, could be useful to have a link to holt winters algorithm here for the sake of reference